src/modules/IPSecConf.rb

```diff
@@ -129,19 +129,19 @@ def GenFirewallScript
             script << func_template % {func_name: "fw_custom_before_port_handling", content: ""}
             # Reduce TCP MSS - if this has to be done, it must come before FORWARD and MASQUERADE
             inet_access = ""
-            if @tcp_mss_1024
-                inet_access += "iptables -A FORWARD -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1024\n" +
-                               "ip6tables -A FORWARD -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1024\n"
+            if @tcp_reduce_mss
+                inet_access += "iptables -A FORWARD -p tcp -m tcp --tcp-flags SYN,RST SYN -m tcpmss --mss 1221:65535 -j TCPMSS --set-mss 1220\n" +
+                               "ip6tables -A FORWARD -p tcp -m tcp --tcp-flags SYN,RST SYN -m tcpmss --mss 1221:65535 -j TCPMSS --set-mss 1220\n"
```

Comment:
This 1221 seems dangerous in case we ever need to change the 1220, because a string search will miss it. Why not define a `MSS_SIZE = 1220` constant and use it everywhere, adding 1 here?